<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EksiXML</class>
 <widget class="QMainWindow" name="EksiXML">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EksiXML</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="openXMLBtn">
          <property name="text">
           <string>Open Ekşi Sözlük XML Backup</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="convertBtn">
          <property name="text">
           <string>Convert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
